tests/dummy/app/routes/application.js

```diff
@@ -1,4 +1,5 @@
 import Ember from 'ember';
+import { getOwner } from '@ember/application';
```

Comment:
Can we go ahead and replace line 1 with `import { inject as service } from '@ember/service';`?

I wasn't sure if you planned to create a separate PR for updating the code to Octane syntax (e.g. using native class instead of `Route.extend()`, etc).

Reply:
Ah, just saw your comment https://github.com/ember-intl/cp-validations/pull/87#issuecomment-1251249744

Reply:
Did it here: https://github.com/ember-intl/cp-validations/pull/90/commits/f03c516e5705da4e51b9955a8a3ca7611acc0790